test/distributed/fsdp/test_fsdp_summon_full_params.py

```diff
@@ -429,7 +429,7 @@ def test_reshard_outside_forward_backward_iteration(
     def test_params_are_unflattenned(self, rank0_only, offload_to_cpu, mixed_precision):
         layer_shape = (10, 12)
         model = nn.Linear(*layer_shape, bias=False).cuda(self.rank)
-        mixed_precision = MixedPrecision() if mixed_precision else None
+        mixed_precision = MixedPrecision(torch.float16, None, None) if mixed_precision else None
```

Comment:
good catch! after mixed precision refactor, these are not mixed precision anymore before this fix